connector/src/main/java/org/geysermc/connector/network/translators/bedrock/BedrockTextTranslator.java

```diff
@@ -38,12 +38,26 @@
     @Override
     public void translate(TextPacket packet, GeyserSession session) {
         if (packet.getMessage().charAt(0) == '.') {
-            ClientChatPacket chatPacket = new ClientChatPacket(packet.getMessage().replace(".", "/"));
+            String message = packet.getMessage().replace(".", "/").trim();
+
+            if (message.length() > 256) {
+                session.sendMessage("Your message is bigger than 256 characters (" + message.length() + ") so it has not been sent.");
```

Comment:
- Multiple lines doing the same thing, can we simplify this/make it a function?
- May be out of the scope of this PR but there is no localization.